src/contexts/CoursesContext.jsx

```diff
@@ -0,0 +1,25 @@
+// ============================================================================================
+// IMPORTS
+// ============================================================================================
+
+import { createContext, useState } from "react";
+import PropTypes from "prop-types";
+import { UserContextProvider } from "./UserContext";
+
+const CoursesContext = createContext(null);
+
+function CoursesContextProvider({ children }) {
+  const [coursesInfo, setCoursesInfo] = useState(null);
+
+  return (
+    <CoursesContext.Provider value={{ coursesInfo, setCoursesInfo }}>
+      {children}
+    </CoursesContext.Provider>
+  );
+}
+
+UserContextProvider.propTypes = {
+  children: PropTypes.object
+};
+
```

Comment:
You are assigning propTypes to UserContextProvider instead of CoursesContextProvider. This seems to be a typo.

Reply:
You are right!